lineman/test/test_main.py

```diff
@@ -0,0 +1,40 @@
+import unittest
+
+import lineman.__main__ as main
+from lineman.version import __version__
+
+class TestHawkEyeInterop(unittest.TestCase):
+
+    def test_proper_keys(self):
+        pre_proc = {
+            'records': {
+            'num_inputed': 0,
+            'num_redcap': 0,
+            'num_valid': 0,
+            'num_invalid': 0,
+            'validated': [],
+            'not_validated': [],
+            'mappings_done': {},
+            },
+            'subject_event_dict': {}
+        }
+        post_proc = {
+            'source': 'lineman_%s' % __version__,
```

Comment:
Yeah good point, that is a bug. I need to create a setting specifically for the template version. I lost sight of that when I put the regular version stuff in.